soapui-installer/src/install4j/SoapUI.install4j

```diff
@@ -1462,6 +1462,9 @@ if (Util.isMacOS()) {
                           <void index="12">
                             <string>-splash:SoapUI-Spashscreen.png</string>
                           </void>
+                          <void index="13">
+                            <string>-Dlog4j2.formatMsgNoLookups=true</string>
+                          </void>

```

Comment:
We should also change length="13" in array tag.

Reply:
Fixed, also in ReadyAPI. Thank you, it's nice catch>